src/main/java/com/github/bibsysdev/nva/cristin/person/model/nva/NvaPerson.java

```diff
@@ -0,0 +1,90 @@
+package com.github.bibsysdev.nva.cristin.person.model.nva;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.ALWAYS;
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+import static com.github.bibsysdev.nva.cristin.person.PersonsWrapper.CONTEXT;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import java.net.URI;
+import java.util.List;
+
+@JsonInclude(ALWAYS)
+@JsonPropertyOrder("@context")
+public class NvaPerson {
+
+    @JsonInclude(NON_NULL)
+    @JsonProperty(CONTEXT)
+    private String context;
+    private URI id;
+    private String type;
+    private List<NvaIdentifier> identifier;
+    private String name;
```

Comment:
We want both firstname and surname in same var?

Reply:
That's what the specification said.